CHANGELOG.md

```diff
@@ -1,36 +1,72 @@
 # Tridactyl changelog
 
-## Release 1.13.0 / Unreleased
+## Release 1.13.0
```

Comment:
Please leave it unreleased until it is released

Reply:
Done :)